config/manifests-ocp/kustomization.yaml

```diff
@@ -0,0 +1,28 @@
+# These resources constitute the fully configured set of manifests
+# used to generate the 'manifests/' directory in a bundle.
+resources:
+- bases/instaslice-operator.clusterserviceversion.yaml
+- ../ocp
+- ../samples
+- ../scorecard
+
+# [WEBHOOK] To enable webhooks, uncomment all the sections with [WEBHOOK] prefix.
```

Comment:
remove the commented out code if not required.

Reply:
I can, but this is just one file of many.  There are lots of files that have chunks of code commented out.  Operator-sdk generates its files like this for inline documentation.

Reply:
Alright, if it is generated that way then it should be fine IMO.